src/com/facebook/buck/ide/intellij/IjProjectPaths.java

```diff
@@ -55,6 +55,21 @@ static String toModuleDirRelativeString(Path path, Path moduleLocationBasePath)
     }
   }
 
+  /**
+   * @param path path to folder.
+   * @param moduleLocationBasePath path to the location of the .iml file.
+   * @return a path, relative to the module .iml file location describing a folder without the
+   *     IntelliJ format.
+   */
+  static String toRelativeString(Path path, Path moduleLocationBasePath) {
+    String moduleRelativePath = moduleLocationBasePath.relativize(path).toString();
+    if (moduleRelativePath.isEmpty()) {
+      return "";
+    } else {
+      return "/" + MorePaths.pathWithUnixSeparators(moduleRelativePath);
```

Comment:
Use File separator instead of `/`

Reply:
I used what is used by ProjectPaths.toModuleDirRelativeString(), which this method is derived from